docs/topics/testing/overview.txt

```diff
@@ -259,6 +259,13 @@ To prevent serialized data from being loaded twice, setting
 :data:`~django.db.models.signals.post_migrate` signal when flushing the test
 database.
 
+.. versionchanged:: 1.10
```

Comment:
I'm not sure, as this logic is still dependant on `serialized_rollback` parameter, as explained on second paragraph:

"Django can reload that data for you on a per-testcase basis by setting the ``serialized_rollback`` option to ``True`` in the body of the ``TestCase`` or ``TransactionTestCase``"

Not really sure if it makes sense to use `serialized_rollback` with `TestCase`, as these tests are always launched before `TransactionTestCase` ones, and so the initial data migrations should not be impacted by `TestCase` tests, isn't it ?

Just wondering, a mix of `TransactionTestCase` tests with `serialised_rollback` to `True` and `False` won't guarantee the `--keepdb` option to work.

If the last test of the suite has no `serialised_rollback` option, then the database will be empty at the end. And as we cannot guarantee test ordering, it means that somebody who wants to reuse the database with `--keepdb` option should absolutely use `serialized_rollback` option to `True` on every TransactionTestCase he is defining...